btk/draw_blends.py

```diff
@@ -237,7 +252,7 @@ def run_single_band(self, blend_catalog, obs_conds, band):
         galaxy_builder = descwl.model.GalaxyBuilder(
             obs_conds, no_disk=False, no_bulge=False, no_agn=False, verbose_model=False
         )
-        pix_stamp_size = np.int(self.stamp_size / self.pixel_scale)
+        pix_stamp_size = np.int(self.stamp_size / obs_conds.pixel_scale)
```

Comment:
to be consistent with naming, we might want to make the argument passed into `run_single_band` be `cutout` instead of `obs_conds`.

Reply:
I thought about that too, it may be clearer.